### ISSUE TITLE
A CNN with explicit norms partitions on norms instead of residual adds

### ISSUE THREAD

**Issue by aidemsined** [tenstorrent/ttnn-visualizer#1993]
`delimitersFor` picks its delimiter set from the whole graph: normalisation ops if the capture contains **any**, residual adds otherwise. That is deliberate — a graph that mixed both would partition inconsistently down its length — but it means the choice is made globally by presence rather than by what the graph looks like, and `batch_norm` / `group_norm` are in the normalisation set.

So a CNN that emits its norms as ops never reaches the residual-add strategy that the convolutional-residual role depends on. Reproduced:

```
conv2d, batch_norm, relu, conv2d, batch_norm, add
```
partitions into `[1,2]` and `[3,4,5]` — cut at each norm — where the intended shape is one residual block. The residual-conv grouping therefore only works on captures whose norms are fused or absent, which is exactly why `resnet50_aug06` works: 159 `conv2d`, 48 `add_`, and no norm op at all (quasar fuses them).

Raised by Copilot on #1992. Not fixed there, because the fix is a strategy decision and there is no capture in the repo to validate it against: every fixture is either a transformer with `layer_norm` or a CNN with no norms. Wants a `batch_norm`-emitting CNN report as a fifth fixture first.

Shape of a fix, for whoever picks it up: choose the delimiter set from the anchors present rather than from norm presence alone — a graph whose anchors are convolutions wants residual boundaries even when it has norms — or partition on residual adds *within* a norm-delimited span when the span's anchor is a convolution.

Note the same report also exercises the role-priority fix from #1992: `conv2d, relu` spans now name themselves after the convolution rather than the activation, so a batch-norm CNN at least labels its fragments correctly even while they stay fragments.
